examples/02_first_level_models/plot_predictions_residuals.py

```diff
@@ -0,0 +1,149 @@
+"""
+Predicted time series and residuals
+===================================
+
+Here we fit a First Level GLM with the `minimize_memory`-argument set to `False`.
+By doing so, the `FirstLevelModel`-object stores the residuals, which we can then inspect.
+Also, the predicted time series can be extracted, which is useful to assess the quality of the model fit.
+"""
+
+
+#########################################################################
+# Import modules
+# --------------
+from nistats.datasets import fetch_spm_auditory
+from nilearn import input_data, image
+import matplotlib.pyplot as plt
+from nilearn import plotting, masking
+from nistats.reporting import get_clusters_table
+from nistats.first_level_model import FirstLevelModel
+import pandas as pd
+import numpy as np
```

Comment:
If you don't mind, could you split the import so that novel users are not overwhelmed by the ten initial lines of imports.